packages/common/src/lib/common/tools.ts

```diff
@@ -2615,11 +2667,13 @@ async function removeIdFromAllEnums(objects, id, allEnums) {
  *
  * @alias parseDependencies
  * @memberof tools
- * @param {string[]|Record<string, string>[]|string|Record<string, string>} dependencies dependencies array or single dependency
- * @returns {Record<string, string>} parsed dependencies
+ * @param dependencies dependencies array or single dependency
+ * @returns parsedDeps parsed dependencies
  */
-function parseDependencies(dependencies) {
-    let adapters = {};
+function parseDependencies(
+    dependencies: string[] | Record<string, string>[] | string | Record<string, string>
```

Comment:
Gotta love a clean and specific API 🙈

Reply:
historically these are the dependencies which were allowed in io-pack at some point